### PR TITLE
Create queueInC.c

### PR DESCRIPTION
**Pull request by mateuseap**

### CODE REVIEW

File: data_structures/queueInC.c [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    Queue *left, *right;
+    int i, c, cont, flag = 1, viagem = 0;
+    int l, m, it;
+    int actual, total = 0, total2 = 0;
+    char side[max];
+
+    scanf("%d", &c);
+    //printf("Casos Teste %d\n", c);
+
+    while(cont < c){
```
Comment (ashwek) [inline data_structures/queueInC.c]: ``cont`` is never initialized

File: data_structures/queueInC.c [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                    total += actual;
+                }
+            }else{
+                while(frontValue(right) != -1 && total+frontValue(right) <= l){
+                    actual = dequeue(right);
+                    //printf("0 - %d\n", actual);
+                    total += actual;
+                }
+            }
+            flag = (++flag)%2;
```
Comment (ashwek) [inline data_structures/queueInC.c]: this is an undefined behavior, and different compilers may sequence the operation differently. _(Warning: multiple unsequenced modifications to 'flag')_

please change it to ``flag = (flag + 1) % 2;``

File: data_structures/queueInC.c (hunk 1 above)
Comment (mateuseap) [reply data_structures/queueInC.c]: For sure, this is a huge mistake. I corrected now this issue initializing my cont as zero.

File: data_structures/queueInC.c (hunk 2 above)
Comment (mateuseap) [reply data_structures/queueInC.c]: I did it, I changed to flag = (flag+1)%2. I didn't knew about this, I always used flag++ and I had zero problems with this.

File: data_structures/queueInC.c [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+    scanf("%d", &c);
+    //printf("Casos Teste %d\n", c);
+
+    while(cont < c){
+        left = create_queue();
+        right = create_queue();
+        scanf("%d %d", &l, &m);
+        for(i = 0; i < m; i++){
+            scanf("%d %[^\n]", &it, side);
```
Comment (ashwek) [inline data_structures/queueInC.c]: please print what input is asked _(in english)_

**Pull request closed without merging**